pootle/core/mixins/treeitem.py

```diff
@@ -17,6 +17,7 @@
 from django.conf import settings
 from django.core.urlresolvers import set_script_prefix
 from django.utils.encoding import force_unicode, iri_to_uri
+from django.db import connection
```

Comment:
Move this one line up. Please follow our styleguide on imports http://docs.translatehouse.org/projects/pootle/en/latest/developers/styleguide.html#imports